torch/_library/infer_schema.py

```diff
@@ -53,6 +53,13 @@ def infer_schema(
         (Tensor x) -> Tensor
     """
     UNKNOWN_MUTATES = "unknown"
+    pf_globals = prototype_function.__globals__
+    # To do this properly we really need `locals()` from the scope of the
+    # prototype_function. Unfortunately there doesn't seem to be any way to get
+    # that consistently.
+    pf_locals = None
```

Comment:
prototype_function is always a global function (i.e. not a nested function). So the locals() for it is actually the same as globals(). Does that sound right?

Reply:
Actually I take that back, it's not always a global function.